tests/e2e/taqueria-plugin-ipfs-pinata.spec.ts

```diff
@@ -1,132 +1,186 @@
 import { exec as exec1 } from 'child_process';
-import fsPromises from 'fs/promises';
 import util from 'util';
-import { generateTestProject } from './utils/utils';
 const exec = util.promisify(exec1);
-import * as contents from './data/help-contents/pinata-contents';
-
-const taqueriaProjectPath = 'scrap/auto-test-ipfs-pinata-plugin';
-
-// %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-// IF RUNNING LOCALLY MAKE SURE YOU ADD A .env FILE WITH YOUR PINATA JWT
-// %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-
-let JWT: string;
-async function configureForTests() {
-	if (process.env.pinataJwtToken) {
-		JWT = process.env.pinataJwtToken;
-		await exec(`unset pinataJwtToken`);
-		await exec(`echo "pinataJwtToken=${JWT}" > ${taqueriaProjectPath}/.env`);
-	}
-	if (process.env.UNLIMITED_PINATA_TOKEN) {
-		JWT = process.env.UNLIMITED_PINATA_TOKEN;
-		await exec(`echo "pinataJwtToken=${JWT}" > ${taqueriaProjectPath}/.env`);
-		await exec(`cat ${taqueriaProjectPath}/.env`);
-	} else {
-		// The .env file should be in the root directory of the taqueria project
-		// this just makes sure it gets into the test directory for use
-		await exec(`cp ../.env ${taqueriaProjectPath}/.env`);
-	}
-}
-
-describe('e2e testing for the IPFS Pinata plugin with no JWT', () => {
-	beforeAll(async () => {
-		await generateTestProject(taqueriaProjectPath, ['ipfs-pinata']);
-		// TODO: This can removed after this is resolved:
-		// https://github.com/ecadlabs/taqueria/issues/528
-		try {
-			await exec(`taq -p ${taqueriaProjectPath}`);
-		} catch (_) {}
-	});
+import { prepareEnvironment } from '@gmrchk/cli-testing-library';
 
-	// Skipping due to help output changing
-	test.skip('ipfs pinata plugin should show the correct help contents', async () => {
-		const publishRun = await exec(`taq --help`, { cwd: taqueriaProjectPath });
-		expect(publishRun.stdout).toBe(contents.helpContentsIPFSPinataPlugin);
-	});
+describe('IPFS Pinata plugin E2E tests for Taqueria CLI', () => {
+	test('publish will offer contextual help', async () => {
```

Comment:
Nitpick: "publish *offers* contextual help"

Reply:
I am using NASA requirement guidelines, which require active will, can, etc.
Each action is followed by a modal verb ...
https://www.nasa.gov/seh/appendix-c-how-to-write-a-good-requirement